LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Not sure about the type of LICENSE. Feel free to change it. I've added the most popular one.